libcontainer/system/linux.go

```diff
@@ -214,3 +214,13 @@ func Copy(dst io.Writer, src io.Reader) (copied int64, err error) {
 fallback:
 	return io.Copy(dst, src)
 }
+
+// SetLinuxPersonality sets the Linux execution personality. For more information see the personality syscall documentation.
+// checkout getLinuxPersonalityFromStr() from libcontainer/specconv/spec_linux.go for type conversion.
+func SetLinuxPersonality(personality int) error {
+	_, _, errno := syscall.Syscall(syscall.SYS_PERSONALITY, uintptr(personality), 0, 0)
```

Comment:
How about ‘unix.Syscall’ and ‘unix.SYS_PERSONALITY’?

Reply:
emm,let me think, means keep the same conversion style with other syscall?

Reply:
https://pkg.go.dev/syscall
Deprecated: this package is locked down. Callers should use the corresponding package in the golang.org/x/sys repository instead. That is also where updates required by new systems or versions should be applied. See https://golang.org/s/go1.4-syscall for more information.

Reply:
SGTM